examples/essa-engine/obj-loader.cpp

```diff
@@ -1,25 +1,26 @@
 #include <EssaEngine/3D/Model.hpp>
 #include <EssaEngine/3D/ObjLoader.hpp>
+#include <EssaEngine/3D/Shaders/Basic.hpp>
+#include <EssaEngine/3D/Shaders/Lighting.hpp>
 #include <EssaUtil/Angle.hpp>
 #include <EssaUtil/Color.hpp>
+#include <LLGL/Core/Transform.hpp>
+#include <LLGL/OpenGL/OpenGL.hpp>
+#include <LLGL/OpenGL/Projection.hpp>
 #include <LLGL/OpenGL/Shader.hpp>
-#include <LLGL/OpenGL/Shaders/Basic330Core.hpp>
-#include <LLGL/OpenGL/Shaders/ShadeFlat.hpp>
-#include <LLGL/OpenGL/Utils.hpp>
-#include <LLGL/Renderer/BatchRenderer.hpp>
-#include <LLGL/Renderer/Renderer.hpp>
-#include <LLGL/Renderer/Transform.hpp>
+#include <LLGL/OpenGL/Transform.hpp>
 #include <LLGL/Window/Keyboard.hpp>
 #include <LLGL/Window/Window.hpp>
 #include <iostream>
 
 int main() {
     llgl::Window window { { 500, 500 }, "OBJ loader", { 3, 2 } };
 
-    llgl::opengl::enable(llgl::opengl::Feature::DepthTest);
-    llgl::opengl::set_clear_color(Util::Color { 255, 128, 128 });
+    // TODO: Port to llgl, especially clear color.
+    glEnable(GL_DEPTH_TEST);
+    glClearColor(1, 0.5, 0.5, 1);
```

Comment:
Use chould not use OpenGL functions directly. Actually I think this can be in wnd constructor